miso-web/src/main/webapp/scripts/box_ajax.js

```diff
@@ -358,14 +358,16 @@ Box.ui = {
   },
 
   exportBox: function(boxId) {
-    Fluxion.doAjax('boxControllerHelperService', 'exportBoxContentsForm', {
-      'boxId': boxId,
-      'url': ajaxurl
-    }, {
-      'doOnSuccess': function(json) {
-        Utils.page.pageRedirect('/miso/download/box/forms/' + json.response);
-      }
-    });
+    jQuery.ajax({
+      url: "/miso/rest/box/" + boxId + "/spreadsheet",
+      type: "GET",
+      contentType: "application/json; charset=utf8",
+      dataType: "json",
```

Comment:
Probably doesn't hurt, but no need to specify datatype since no data is being sent

Reply:
I think `dataType` is the type of data being sent back to the client, but I will verify.

Reply:
You're right - I had `dataType` and `contentType` mixed up. `contentType` is the one that should match the accompanying `data`

Reply:
The call should be similar to the other AJAX calls in this file, but everything works without `contentType` and without `dataType`.